lib/her/model/relation.rb

```diff
@@ -64,11 +64,34 @@ def kind_of?(thing)
       #
       # @private
       def fetch
+        # Rails.logger.debug "**** Her::Model::Relation fetch"
+        # Rails.logger.debug "**** Her::Model::Relation fetch @params: #{@params}"
+
         @_fetch ||= begin
           path = @parent.build_request_path(@params)
           method = @parent.method_for(:find)
+
+          # Translate page and per_page params to params understood by the service
+          pagination_param_map = {"page" => "page[number]", "per_page" => "page[size]"}
+          @params = @params.transform_keys {|key| pagination_param_map[key.to_s] || key }
+
           @parent.request(@params.merge(:_method => method, :_path => path)) do |parsed_data, response|
-            @parent.new_collection(parsed_data)
+
+            # Rails.logger.debug "**** Her::Model::Relation @parent: #{@parent}"
+            # Rails.logger.debug "**** Her::Model::Relation @params: #{@params}"
+            # Rails.logger.debug "**** Her::Model::Relation response: #{response.inspect}"
+            # Rails.logger.debug "**** Her::Model::Relation parsed_data: #{parsed_data}"
+            # Rails.logger.debug "**** Her::Model::Relation LINKS: #{parsed_data.links}"
+
+            # Rails.logger.debug "**** Her::Model::Relation LINKS: parsed_data.links.nil? #{parsed_data.links.nil?}"
+            # Rails.logger.debug "**** Her::Model::Relation LINKS: parsed_data.links.size #{parsed_data.links.size}"
+
+            unless parsed_data.links.nil?
+              @parent.new_paginated_collection(parsed_data, @params, response)
+            else
+              @parent.new_collection(parsed_data)
+            end
```

Comment:
I think it would be a little easier to read reversed:
```
if parsed_data.links.nil? # No links means no pagination
  @parent.new_collection(parsed_data)
else
  @parent.new_paginated_collection(parsed_data, @params, response)
end
```